front/src/components/boxs/cinema/style.css

```
@@ -0,0 +1,70 @@
+@keyframes skeletonPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "skeletonPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)